heron/simulator/src/java/shade.conf

```diff
@@ -0,0 +1,18 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Heron API is included in this build which has this shade. I think it is safer to apply the same process.

I think the shade is used to avoid conflicts if different version of the libraries are used in the user topology.